cmd/create/execution_util.go

```diff
@@ -52,7 +52,7 @@ func createExecutionRequestForWorkflow(ctx context.Context, workflowName, projec
 		}
 	}
 
-	return createExecutionRequest(lp.Id, inputs, securityContext, authRole, targetExecName), nil
+	return createExecutionRequest(lp.Id, inputs, securityContext, authRole, targetExecName, executionConfig.OverwriteCache), nil
```

Comment:
Does this need to be passed as an argument? IIUC the `executionConfig` variable is populated before this is called and so it is available in the `createExecutionRequest` - [example here](https://github.com/flyteorg/flytectl/blob/209fe933ac228d23dab89bb6decba587a2b42206/cmd/create/execution_util.go#L157-L158).

Reply:
Good point, yes, seems I've missed that. Will fix shortly!